core/src/main/scala/org/apache/spark/TestUtils.scala

```diff
@@ -255,6 +255,19 @@ private[spark] object TestUtils {
     attempt.isSuccess && attempt.get == 0
   }
 
+  def isPythonVersionAtLeast38(): Boolean = {
+    val attempt = if (Utils.isWindows) {
+      Try(Process(Seq("cmd.exe", "/C", "python3 --version"))
+        .run(ProcessLogger(_ => ()))
```

Comment:
Why do we don't need to check the output from Python execution in the Window's case?

Reply:
Oops.